cmd/launcher/desktop/desktop.go

```diff
@@ -49,6 +57,7 @@ func RunDesktop(args []string) error {
 		systray.AddSeparator()
 		systray.AddMenuItem("--- DEBUG ---", "").Disable()
 		systray.AddMenuItem(fmt.Sprintf("Hostname: %s", *flhostname), "").Disable()
+		systray.AddMenuItem(fmt.Sprintf("Socket Path: %s", desktop.DesktopSocketPath(os.Getpid())), "").Disable()
```

Comment:
Probably don't want to keep this long term. Seems fine for now

Reply:
removed